src/main/resources/changelog/changes/add-source-to-role-and-policy-schemas.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+               http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+  <changeSet id="add source field to role and policy schemas" author="Saba Zedginidze">
+    <sql>
+      DO '
+      BEGIN
+        IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = ''source_type'') THEN
+          CREATE TYPE source_type AS ENUM (''SYSTEM'', ''USER'', ''CONSORTIUM'');
+        END IF;
+      END ' ;
+    </sql>
+
+    <addColumn tableName="policy">
+      <column name="source" type="source_type"/>
```

Comment:
is it possible to have null values in the field?
* if yes, what is the reason for that decision? 
* if no, I don't see a default value specified neither in the sql scripts nor in the code

Reply:
Should not be null, modified

Reply:
as a second option we can set like this `<constraints nullable="false"/>`